gulpfile.babel.js

```diff
@@ -154,6 +154,18 @@ gulp.task('html', () => {
       removeStyleLinkTypeAttributes: true,
       removeOptionalTags: true
     })))
+    .pipe($.if('*.html', $.minifyInline({
+            js: {
+                output: {
+                    comments: true
+                }
+            },
+            jsSelector: 'script[type!="text/x-handlebars-template"]',
```

Comment:
As far as I know, we don't use handlebars in the project. We shouldn't add exclusions to things not used in the repo.